### ISSUE TITLE
Port tool-bar-get-system-style

### ISSUE THREAD

**Issue by brotzeit** [remacs/remacs#1461]
```c
DEFUN ("tool-bar-get-system-style", Ftool_bar_get_system_style,
       Stool_bar_get_system_style, 0, 0, 0,
       doc: /* Get the system tool bar style.
If no system tool bar style is known, return `tool-bar-style' if set to a
known style.  Otherwise return image.  */)
  (void)
{
  if (EQ (Vtool_bar_style, Qimage)
      || EQ (Vtool_bar_style, Qtext)
      || EQ (Vtool_bar_style, Qboth)
      || EQ (Vtool_bar_style, Qboth_horiz)
      || EQ (Vtool_bar_style, Qtext_image_horiz))
    return Vtool_bar_style;
  if (!NILP (current_tool_bar_style))
    return current_tool_bar_style;
  return Qimage;
}
```

**Comment by Bira** [comment]
Since I recently merged ports for a couple of other functions from the same file, I'd like to take this one as well. Is that OK?

**Comment by Bira** [comment]
Resolved by #1487 .
